tests/checkpoint/conftest.py

```diff
@@ -25,6 +25,36 @@ def titanic_pandas_data_context_stats_enabled_and_expectation_suite_with_one_exp
     return context
 
 
+@pytest.fixture
```

Comment:
We might embellish this fixture later (e.g., if additional Fluent Datasources and/or DataAssets are needed).